src/ontology/oeo.omn

```diff
@@ -453,3 +456,17 @@ Class: <http://purl.obolibrary.org/obo/BFO_0000038>
 Class: <http://purl.obolibrary.org/obo/BFO_0000141>
 
     
+Class: Model
+
+    
+Class: ModelComponent
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A ModelComponent is a generically dependent continuant that is part of a model.",
+        rdfs:label "Model component",
+        rdfs:label "Model element"
```

Comment:
Rather than having two labels, I would consider having one primary label and have the synonym have annotation type 'alternative term' (http://purl.obolibrary.org/obo/IAO_0000118). 

Also, I think perhaps these annotations should be asserted in oeo-model rather than oeo.omn?